flexget/components/emby/api_emby.py

```diff
@@ -242,7 +243,10 @@ def login(self, optional=False):
                             raise PluginError(
                                 f'Could not login to Emby Connect account `{self._connect_username}`, no server list'
                             )
-                        if server['Name'].lower() == self.host.lower():
+                        if (
+                            server['Name'].lower() == self.host.lower()
+                            or server['Name'].lower() == self.host_name.lower()
```

Comment:
What are the cases when `self.host_name` is different than `self.host`? It looks like they just get set the same down on line 358.

Reply:
When the user sets the config, the host can a address, when we are connecting directly to the server, or a name, that is used to match the server name in emby connect (a proxy hosted by the emby team). To make the code readable, I replace the emby name with the actual address (supplied by emby connect on login), but with that I need to save the server name to use latter on, for example on the reconnect! That was the main issue this PR was solving!